src/app/App.tsx

```diff
@@ -1,6 +1,34 @@
 import React from 'react';
+import styles from './App.module.css';
+import Title from './components/title/Title';
 
 function App(): JSX.Element {
-  return <main>Hello Fishes!</main>;
+  return (
+    <main className={styles.container}>
+      <img
+        className={styles.image}
+        src="src/app/assets/people-partying-close-up-photography-2705089.jpg"
+        alt="Some people chilling on a table"
+      />
+      <div>
+        <Title />
+        <form className={styles.form}>
+          <input
+            className={styles.input}
+            type="text"
+            placeholder="First name"
+          />
+          <input className={styles.input} type="text" placeholder="Last name" />
+          <input
+            className={styles.submitbutton}
```

Comment:
```suggestion
            className={styles.submitButton}
```

Like already suggested in the module.css file 🔢